content/using_bookwyrm/posting-statuses.md

```diff
@@ -26,6 +26,17 @@ There are three types of posts users can create directly in response to books: r
 | Rating | ✔ |  |  |
 | Title field | ✔ |  |  |
 
+#### Spoiler alerts
+
+Spoiler alerts (also known as content warnings) are useful to give people a warning before they read a status that might spoil the plot of a book they want to read.
+Usually just putting "contains spoilers" or something similar is sufficient, but you can be more specific, especially if your status discusses potentially sensitive topics.
+
+#### Star ratings
+
+A star rating can be added to reviews, or added on it's own by clicking the stars below a book.
```

Comment:
Should be `or added on its own` (no possessive apostrophe for its, apostrophes in English are hard...).